kubernetes/deployments_kubeclient.go

```diff
@@ -1685,6 +1694,10 @@ func (kc *kubeClient) deleteRoutes(appName string, envNS string) error {
 	// Delete all dependent objects before deleting the route
 	policy := metaV1.DeletePropagationForeground
 	opts := &metaV1.DeleteOptions{
+		TypeMeta: metaV1.TypeMeta{ // Normally set automatically by k8s client-go
```

Comment:
This looks like it's either old code, or belongs in a separate PR.

Reply:
See above.